recipes/conversation_assistant/conversation.js

```diff
@@ -60,34 +126,26 @@ console.log("Say 'stop' or press ctrl-c to exit this recipe.");
 // listen for utterances with our attentionWord and send the result to
 // the Assistant service
 while (true) {
-    const msg = await tj.listen();
-
-    if (msg === 'stop') {
-        console.log('Goodbye!');
-        process.exit(0);
-    }
-
-    // check to see if they are talking to TJBot
-    if (msg.toLowerCase().startsWith(config.robotName.toLowerCase())) {
-        // remove our name from the message
-        const utterance = msg.toLowerCase().replace(config.robotName.toLowerCase(), '').substr(1);
-        // const utterance = msg.toLowerCase();
+const msg = await tj.listen();
 
-        // send to the assistant service
-        const response = await tj.converse(utterance);
-        let spoken = false;
+if (msg === 'stop') {
+    console.log('Goodbye!');
+    process.exit(0);
+}
 
-        // check if an intent to control the bot was found
-        if (response.object.intents !== undefined) {
-            console.log(`found intent(s): ${JSON.stringify(response.object.intents)}`);
+// check to see if they are talking to TJBot
+if (msg.toLowerCase().startsWith(config.robotName.toLowerCase())) {
```

Comment:
This line has been tricky in the past… on one hand, you don't want tjbot responding to absolutely everything it hears. But on the other, you want a wake-word that is easily detectable. I think we struggled in the past with "Watson".

Reply:
Would you suggest a change to the code here or changing the default robot name to something else? In testing it works fine but obviously that's via text only not audio.